src/main/java/ui/UiRenderer.java

```diff
@@ -104,54 +104,54 @@ public static void printTasksInWeek(LocalDate startOfWeek, TaskManager taskManag
         int maxNumberOfLinesPerTask = getMaxNumberOfLinesPerTask(allWrappedTaskLines);
         printTasksInGrid(startOfWeek, maxNumberOfTasksInDay, maxNumberOfLinesPerTask);
     }
-    
+
     /**
      * Returns the maximum number of tasks in a day.
-     * 
+     *
      * @param allWrappedTaskLines The map of wrapped task lines.
      * @return The maximum number of tasks in a day.
      */
     private static int getMaxNumberOfTasksInDay(Map<LocalDate, List<List<String>>> allWrappedTaskLines) {
         int maxNumberOfTasksInDay = 0;
-        
+
         for (List<List<String>> dayTasks : allWrappedTaskLines.values()) {
             if (dayTasks.size() > maxNumberOfTasksInDay) {
                 maxNumberOfTasksInDay = dayTasks.size();
             }
         }
-        
+
         return maxNumberOfTasksInDay;
     }
 
     /**
      * Returns the maximum number of lines per task.
-     * 
+     *
      * @param allWrappedTaskLines The map of wrapped task lines.
      * @return The maximum number of lines per task.
      */
     private static int getMaxNumberOfLinesPerTask(Map<LocalDate, List<List<String>>> allWrappedTaskLines) {
         int maxNumberOfLinesPerTask = 0;
-        
+
         for (List<List<String>> dayTasks : allWrappedTaskLines.values()) {
             for (List<String> taskLines : dayTasks) {
                 if (taskLines.size() > maxNumberOfLinesPerTask) {
                     maxNumberOfLinesPerTask = taskLines.size();
                 }
             }
         }
-        
+
         return maxNumberOfLinesPerTask;
     }
-    
+
     /**
      * Prints the tasks in a grid format for the week. The tasks across different days are aligned in the same row.
-     * 
+     *
      * @param startOfWeek The date of the start of the week.
      * @param maxNumberOfTasksInDay The maximum number of tasks in a day.
      * @param maxNumberOfLinesPerTask The maximum number of lines per task.
      */
-    private static void printTasksInGrid(LocalDate startOfWeek, int maxNumberOfTasksInDay, 
-            int maxNumberOfLinesPerTask) {
+    private static void printTasksInGrid(LocalDate startOfWeek, int maxNumberOfTasksInDay,
+                                         int maxNumberOfLinesPerTask) {
```

Comment:
Would the original indentation be more appropriate?

Reply:
Likely so. I think this is a code checkstyle violation - [see indentation for wrapped lines](https://se-education.org/guides/conventions/java/basic.html)